packages/zoe/src/objArrayConversion.js

```diff
@@ -57,6 +60,7 @@ export const assertSubset = (whole, part) => {
  * @returns {Pick<T,U[number]>}
```

Comment:
What's `Pick`?

Reply:
Create a record whose keys keys are elements of the `U` tuple (fixed-length array) and whose values are effectively `{U: T[U]}`.

https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys